wgsl/index.bs

```diff
@@ -1672,6 +1672,11 @@ Extension names are not [=identifiers=]: they do not [=resolves|resolve=] to [=d
       <td>
       Removes address space and memory view restrictions on pointer parameters
       of user-defined functions
+  <tr><td><dfn noexport dfn-for="language-extension">packed_4x8_integer_dot_product</dfn>
```

Comment:
Since the built-in functions don't need to describe the requirement for use (see later comment), this definition could be dropped.